### ISSUE TITLE
Package name and suggested shortcut

### ISSUE THREAD

**Issue by navidcy** [COSIMA/regional-mom6#33]
What should we suggest as the shortcut?

This is, of course, a suggestion but some of these become staples. E.g., `np`, `plt`, ...

Suggestions:

```python
import mom6_regional as mom6r
```

```python
import mom6_regional as mr
```

```python
import mom6_regional as rm
```

```python
import mom6_regional as m6r
```

?

**Comment by navidcy** [comment]
I may even suggest a rename of `mom6-regional` to `regional-mom6`? This is how we'd say it in English... 

"I installed the regional mom6 package and with that I set up a configuration in no time!"

"I installed the mom6 regional package and with that I set up a configuration in no time!"

**Comment by navidcy** [comment]
After personal communication w @ashjbarnes we seemed to be converging to

`regional-mom6`

for package name + repo name, and 

```python
import regionan-mom6 as rm
```
for the suggested shortcut.

Any ideas, objections, suggestions?

cc @PaulSpence @ChrisC28 @angus-g @AndyHoggANU @aekiss

**Comment by angus-g** [comment]
I don't have any other suggestions for the shortcut, but I think the repo name would be `regional-mom6` and then the package/files would be `regional_mom6` (for technical reasons).

**Comment by navidcy** [comment]
Yeah... Damn those technicalities.
What are the allowed characters for python package names? Where can we find the "python constitution"?

**Comment by angus-g** [comment]
We're *allowed* to have italics, but the convention is to use underscores for things you import (because you can't `import regional-mom6`, it interprets it as a subtraction...). I think the relevant PEP 8 guideline is: https://peps.python.org/pep-0008/#package-and-module-names

**Comment by navidcy** [comment]
same issue with `cosima_cookbook`...

then, `regional_mom6` or `regionalmom6`. I think the former is more readable...

**Comment by angus-g** [comment]
I think `regional_mom6` is preferable to `regionalmom6`, personally!

**Comment by ashjbarnes** [comment]
I think that regional-mom6 is nice, and `import as rm` personally!

**Comment by navidcy** [comment]
ok, I'll make the change and the rest people tagged here better speak soon or forever hold their peace ❤️ :)

**Comment by navidcy** [comment]
Closed w #47
